samples/bluetooth/mesh/light/README.rst

```diff
@@ -44,6 +44,10 @@ Provisioning is performed using the `nRF Mesh mobile app`_.
 This mobile application is also used to configure key bindings, and publication and subscription settings of the Bluetooth mesh model instances in the sample.
 After provisioning and configuring the mesh models supported by the sample in the `nRF Mesh mobile app`_, you can control the LEDs on the development kit from the app.
 
+.. note::
+   The sample does not publish any status messages at the end of transitions, which in turn may result in invalid behavior in applications modeled on this sample.
+   Make sure to implement the end-of-transition publication for your application.
```

Comment:
This is quite confusing. This seems like telling users that samples have invalid behavior(?). I think that is not correct. A small part of the behavior according to the spec recommendation is not modeled.

```suggestion
   The sample does not publish any status messages at the end of transitions, however, Mesh Model specification recommends that a status message is published at the of the transitions.
   Make sure to implement the end-of-transition publication for your application.
```